elasticgraph-graphql/spec/acceptance/search_spec.rb

```diff
@@ -310,6 +310,28 @@ module ElasticGraph
           string_hash_of(widget3, :id, :amount_cents)
         ])
 
+        # Test `not: {any_of: []}` results in all matching documents
+        widgets = list_widgets_with(:amount_cents,
+          filter: {not: {any_of: []}})
+
+        expect(widgets).to match([
+          string_hash_of(widget3, :id, :amount_cents),
+          string_hash_of(widget2, :id, :amount_cents),
+          string_hash_of(widget1, :id, :amount_cents)
+        ])
+
+        # Test `not: {any_of: [emptyPredicate]}` results in no matching documents
+        widgets = list_widgets_with(:amount_cents,
+          filter: {not: {any_of: [{name: {equal_to_any_of: nil}}]}})
+
+        expect(widgets).to eq []
+
+        # Test `not: {any_of: [emptyPredicate && nonEmptyPredicate]}` results in no matching documents
```

Comment:
I brought this up in https://github.com/block/elasticgraph/pull/6#discussion_r1835227256 but `any_of: [emptyPredicate && nonEmptyPredicate]` is quite confusing because `any_of` is an OR but `&&` commonly means `AND`.

Can this be `any_of: [emptyPredicate, nonEmptyPredicate]`?

Reply:
Thanks, must have forgotten to change from `&&` to `,`